solidity/contracts/converter/ConverterUpgrader.sol

```diff
@@ -5,6 +5,16 @@ import "./interfaces/IConverterUpgrader.sol";
 import "./interfaces/IConverterFactory.sol";
 import "../utility/ContractRegistryClient.sol";
 
+interface IConverterWithTokenHolder is IConverter {

```

Comment:
can just rename to ILegacyConverter in case we need to add more interfaces there.

Reply:
👍